assets/flake.hbs

```diff
@@ -0,0 +1,55 @@
+{
+  {{#if description}}
+  description = "{{ description }}";
+  {{/if}}
+
+  inputs = {
+    {{#each inputs}}
+    {{@key}}.url = "{{this}}";
+    {{/each}}
+  };
+
+  outputs = { self{{#each inputs}}, {{@key}}{{/each}} }:
+    let
+      {{#if has_overlays}}
+      overlays = [
+        {{#each overlay_refs}}
+        {{this}}
+        {{/each}}
+        {{#if overlay_attrs}}
+        (final: prev: {
+          {{#each overlay_attrs}}
+          {{@key}} = {{this}};
+          {{/each}}
+        })
+        {{/if}}
+      ];
+      {{/if}}
+      supportedSystems = [ {{#each systems}}"{{ this }}"{{#unless @last}} {{/unless}}{{/each}} ];
+      forEachSupportedSystem = f: nixpkgs.lib.genAttrs supportedSystems (system: f {
+        pkgs = import nixpkgs { inherit {{#if has_overlays }}overlays {{/if}}system; };
```

Comment:
Would it be ideal to always use nixpkgs with overlays? I'm asking because it might increase the eval time, for reference: https://zimbatm.com/notes/1000-instances-of-nixpkgs